CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ All notable changes to this project will be documented in this file.
 -   #42 : Add support for custom kernel in`cuda`.
 -   #42 : Add Cuda module to Pyccel. Add support for `cuda.synchronize` function.
 -   #41 : Add support for custom device in`cuda`.
+-   #69 : Improve kernel decorator.
```

Comment:
This should go in the Changed section not the Added section, but I'm not actually sure that it needs to be here at all as the Change Log describes things that have changed since the previous release

Reply:
[aa76f91](https://github.com/pyccel/pyccel-cuda/pull/69/commits/aa76f916848235c095e44be7c820d496cec000a5)